ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/questionnaire-completed/questionnaire-completed.component.html

```diff
@@ -3,12 +3,116 @@
     <h1 class="govuk-heading-xl" i18n="@@questionnaireCompleted_h1">
       Questionnaire completed
     </h1>
+    <div>
+      <app-hearing-details-header></app-hearing-details-header>
+    </div>
+    <div>
+      <p class="govuk-body">Here's a summary of the information you've provided.</p>
+    </div>
+
+    <h2 class="govuk-heading-m">About you</h2>
+    <table class="govuk-table" style="width:100%;">
```

Comment:
When I did the header details component I used a css class for the width instead since using inline styles makes it impossible to override them using css. What do you think?
```
.vh-fullwidth {
    width:100%;
}
```

Reply:
Good point but I have removed this as I dont need it, the .govuk-table has a full width 100%